ui-modules/blueprint-composer/app/components/providers/palette-service.provider.js

```diff
@@ -37,7 +37,9 @@ export function paletteServiceProvider() {
             sections[id] = section;
         },
         deleteSection(id) {
+            let old = sections[id];
             delete sections[id];
+            return old;
```

Comment:
I'm curious, why returning `old`?

Reply:
if you want to change order a common pattern would be to delete it but put it to a local var so you could then add it in the right place.  same idea as `Map.remove`